src/_data/LiveData.js

```diff
@@ -259,7 +259,6 @@ export const setAccountCurrency = (currency, store) => {
       if (r.set_account_currency === 1) {
         const loginid = state.account.toJS().loginid;
         addCurrencyToAccount(currency, loginid);
-       window.location = window.BinaryBoot.baseUrl;
```

Comment:
@nazaninreihani I've shifted the refresh page code to `SetCurrency.js` instead, since I can't access the router from there.

Reply:
@brucebinary users can choose currency in upgrade to maltainvest process and after upgrading success a request to set currency is sent, are you sure in that case it will work too? this process is not related to set currency page

Reply:
@brucebinary the process is like this =>
please choose account currency in 'create new account' in user accounts section and go to upgrade (upgrade via maltainvest account opening), then upgrade, this should work correct, I'll test it too tommorow